Dev

hyperfree opened the pull request:
Added "news sites" as resource category incl bitcoinnews.com lightning.news & btctimes.com. Need review and update of IDs, unsure how they are created.

LoicPandul committed:
Update script tutorial creation (#1444)

* Update new-tutorial-creation.py

* Update new-tutorial-creation.py

* Update new-tutorial-creation.py

* Update README.md

* Update new-tutorial-creation.py

* Update new-tutorial-creation.py

* Update README.md

* Update new-tutorial-creation.py

Asi0Flammeus committed:
prof, update name

Asi0Flammeus committed:
courses, add published_at field

Asi0Flammeus committed:
courses, add published_at field

Commit:
adding_hindi as a language (#1418)

* adding_hindi

* Update supported_languages.json

---------

Co-authored-by: Mari <Mari@jjhodl.com>
Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

Pivii committed:
[TUTORIAL] New Bisq2 Tutorial  (#1423)

* Create folder, and update fr.md

* update fr.md

* add iamges

* add images and update fr.md

* paiement info hidden and update fr.md

* Delete settings.json

* Delete logo.webp

* Update fr.md

* Update tutorial.yml

---------

Co-authored-by: Pierre <pierre.vvr@protonmail.com>
Co-authored-by: Loïc Morel <137194052+LoicPandul@users.noreply.github.com>
Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

LoicPandul committed:
Fix links to tutorials (#1433)

* new Relai tutorial link

* new LNP2PBOT tutorial link

* BTC102 links ru.md

* Update ru.md

* Update es.md

alevandx committed:
alevand-professor-update-1 (#1434)

I updated my information (twitter, description and donation link) and added more languages

LoicPandul committed:
[BUILDER] Add Canonical (#1435)

* Create logo.webp

* Create builder.yml

* Create cs.yml

* Create de.yml

* Create en.yml

* Create es.yml

* Create et.yml

* Create fi.yml

* Create fr.yml

* Create nb-NO.yml

* Create vi.yml

* Create it.yml

* Create ja.yml

* Create pt.yml

* Create id.yml

* Create ru.yml

* Create zh-Hans.yml

BitPreto committed:
[REOSURCE] Fix Zum Wilden Michel Farm builder category (#1438)

networkeffect committed:
Modifs additionnelles biz101 (#1439)

* Update en.md

* Update 07.webp

* Update en.md

* Update en.md

* Update en.md

* Update en.md

---------

Co-authored-by: Loïc Morel <137194052+LoicPandul@users.noreply.github.com>

LoicPandul committed:
[TUTORIAL] Edit Peach (#1441)

* add french youtube tuto on fr.md

* remove old tuto peach

* rename folder -> just "peach"

---------

Co-authored-by: Asi0Flammeus <asi0.flammeus@pm.me>

Asi0Flammeus committed:
tutorial, remove past peach wallet

LoicPandul committed:
[TUTORIAL] Move Braiins mini miner (#1442)

* move tuto from wallet to mining

* BTC101_Final text review and new images (#1354)

* Text review according to new images

* correcting_missing_image

* Adding_missing_descriptions

* Changing_image_names

* last_image

---------

Co-authored-by: Mari <Mari@jjhodl.com>
Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

* event: change format

* builders, add missing ids

* add missing project_id

* proofreading, replace empty field by default value

* proofreading, update to dollar unit

* move projectId above proofreading fields

* proofreading, update unit in dollars

* rename project_id

* chore: update lockfile

* Update script tutorial creation (#1444)

* Update new-tutorial-creation.py

* Update new-tutorial-creation.py

* Update new-tutorial-creation.py

* Update README.md

* Update new-tutorial-creation.py

* Update new-tutorial-creation.py

* Update README.md

* Update new-tutorial-creation.py

* prof, update name

* courses, add published_at field

* courses, add published_at field

* adding_hindi as a language (#1418)

* adding_hindi

* Update supported_languages.json

---------

Co-authored-by: Mari <Mari@jjhodl.com>
Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

* [TUTORIAL] New Bisq2 Tutorial  (#1423)

* Create folder, and update fr.md

* update fr.md

* add iamges

* add images and update fr.md

* paiement info hidden and update fr.md

* Delete settings.json

* Delete logo.webp

* Update fr.md

* Update tutorial.yml

---------

Co-authored-by: Pierre <pierre.vvr@protonmail.com>
Co-authored-by: Loïc Morel <137194052+LoicPandul@users.noreply.github.com>
Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

* Fix links to tutorials (#1433)

* new Relai tutorial link

* new LNP2PBOT tutorial link

* BTC102 links ru.md

* Update ru.md

* Update es.md

* alevand-professor-update-1 (#1434)

I updated my information (twitter, description and donation link) and added more languages

* [BUILDER] Add Canonical (#1435)

* Create logo.webp

* Create builder.yml

* Create cs.yml

* Create de.yml

* Create en.yml

* Create es.yml

* Create et.yml

* Create fi.yml

* Create fr.yml

* Create nb-NO.yml

* Create vi.yml

* Create it.yml

* Create ja.yml

* Create pt.yml

* Create id.yml

* Create ru.yml

* Create zh-Hans.yml

* [REOSURCE] Fix Zum Wilden Michel Farm builder category (#1438)

* Modifs additionnelles biz101 (#1439)

* Update en.md

* Update 07.webp

* Update en.md

* Update en.md

* Update en.md

* Update en.md

---------

Co-authored-by: Loïc Morel <137194052+LoicPandul@users.noreply.github.com>

* [TUTORIAL] Edit Peach (#1441)

* add french youtube tuto on fr.md

* remove old tuto peach

* rename folder -> just "peach"

---------

Co-authored-by: Asi0Flammeus <asi0.flammeus@pm.me>

* tutorial, remove past peach wallet

* move tuto from wallet to mining

---------

Co-authored-by: MarJJ <mariannacon@protonmail.com>
Co-authored-by: Mari <Mari@jjhodl.com>
Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>
Co-authored-by: trigger67 <10185784+trigger67@users.noreply.github.com>
Co-authored-by: Asi0Flammeus <asi0.flammeus@pm.me>
Co-authored-by: Pierre <123646307+Pivii@users.noreply.github.com>
Co-authored-by: Pierre <pierre.vvr@protonmail.com>
Co-authored-by: Alevand <166409353+alevandx@users.noreply.github.com>
Co-authored-by: Bitpreto <50926138+PaoloFoti@users.noreply.github.com>
Co-authored-by: NetworkEffect <elie@networkeffect.fr>

Asi0Flammeus committed:
fix various sync issues

Asi0Flammeus committed:
btc204, update zh-hans version

LoicPandul committed:
[TUTORIAL] - Add kraken pro (#1456)

* Create cover.webp

* Create fr.md

* Create tutorial.yml

* txt Kraken pro OK

* img

* img path + rognage

* Update fr.md

trigger-xyz committed:
rename builder folder to project

trigger-xyz committed:
Merge pull request #1458 from PlanB-Network/rename-builder

rename builder folder to project

trigger-xyz committed:
rename builder.yml to project.yml

Asi0Flammeus committed:
tutorials, add automatic translations

LoicPandul committed:
Delete hours.py (#1468)

BitPreto committed:
[RESOURCE] Add & Edit Books (#1457)

* books, newly added

* book, edit descriptions

* Update book.yml

* Update book.yml

* Update book.yml

* Update book.yml

* Update ja.yml

---------

Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

LoicPandul committed:
[TUTORIAL] Add Bitfinex corpo (#1459)

* Create tutorial.yml

* Create cover.webp

* txt tuto ok

* orthographe

* Update fr.md

* img + paths

* rognage

LoicPandul committed:
add links to tutos kraken and bitfinex corpo version (#1460)

LoicPandul committed:
[SCRIPT] Update tutorial creation (#1461)

* builder -> project

* Update new-tutorial-creation.py

* add tags rules

* tags rules

* Clear button confirmation before proceed

Pivii committed:
[TUTORIAL] Ubuntu Installation Guide (#1462)

* Add Ubuntu tutorial files

* Add comprehensive French Ubuntu tutorial with images

* Enhance Ubuntu French tutorial with detailed installation steps and screenshots

* Add additional French Ubuntu tutorial screenshots

* Add ProtonMail recommendation to Ubuntu French tutorial conclusion

* Update image paths to use assets directory

* Add project_id to Ubuntu tutorial configuration

* Update tags

* Size fix

* Fix Loïc feedbacks

---------

Co-authored-by: Pierre <pierre.vvr@protonmail.com>

jramos0 committed:
New International events (#1463)

* BTC in DC project & conference

* Swiss Conference 2025

* Le Paradigma Bitcoin 2025

* Update event.yml

---------

Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

LoicPandul committed:
Update Loïc prof profile (#1464)

* Update profile.webp

* Update fr.yml

* Update fr.yml

* bio

LoicPandul committed:
[TUTORIAL] Edit Bitfinex individual (#1466)

* new txt

* Update tutorial.yml

* new img

* delete translations

* Delete 01.webp

Asi0Flammeus committed:
update id for btc-in-dc

Asi0Flammeus committed:
courses, replace wizard to expert level

midimarcus committed:
WIP: initial creation for Laura Lionti's profile (#1474)

Asi0Flammeus committed:
events, remove project_id

Asi0Flammeus committed:
add uuid v4 for all content type

Asi0Flammeus committed:
update gitignore

Asi0Flammeus committed:
prof, update

Asi0Flammeus committed:
Fix path translation (#1484)

* scripts, use Path lib

* scripts, fix relative file path

* auto-translate, update requirements

* scripts, fix relative file path for tutos

* scripts, fix relative file path for resources

* scripts, fix relative file path for single course

* courses, add missing translations

---------

Co-authored-by: Asi0Flammeus <datascript@ns3104763.ip-145-239-142.eu>

Asi0Flammeus committed:
quizz, remove additional : character

Asi0Flammeus committed:
courses, add new translations

Asi0Flammeus committed:
translation, update supported languages

Asi0Flammeus committed:
add google translation provider (#1490)

LoicPandul committed:
add lipa project (#1472)

Pivii committed:
[TUTORIAL] Lipa Wallet User Guide (#1486)

* Add Lipa Lightning wallet tutorial

* Update fr.md

* Update fr.md

* Add ressources

* Update tutorials/wallet/lipa/fr.md

Co-authored-by: Loïc <137194052+LoicPandul@users.noreply.github.com>

* Update tutorials/wallet/lipa/fr.md

Co-authored-by: Loïc <137194052+LoicPandul@users.noreply.github.com>

* Update tutorials/wallet/lipa/fr.md

Co-authored-by: Loïc <137194052+LoicPandul@users.noreply.github.com>

* Update tutorials/wallet/lipa/fr.md

Co-authored-by: Loïc <137194052+LoicPandul@users.noreply.github.com>

* Update tutorials/wallet/lipa/fr.md

Co-authored-by: Loïc <137194052+LoicPandul@users.noreply.github.com>

* Update tutorials/wallet/lipa/fr.md

Co-authored-by: Loïc <137194052+LoicPandul@users.noreply.github.com>

* Update tutorials/wallet/lipa/fr.md

Co-authored-by: Loïc <137194052+LoicPandul@users.noreply.github.com>

* Update tutorials/wallet/lipa/tutorial.yml

Co-authored-by: Loïc <137194052+LoicPandul@users.noreply.github.com>

* Images cropped

---------

Co-authored-by: Pierre <pierre.vvr@protonmail.com>
Co-authored-by: Loïc <137194052+LoicPandul@users.noreply.github.com>

LoicPandul committed:
[TUTORIAL] Add Jade Plus - Sparrow Wallet (QR mode) (#1473)

* Create cover.webp

* Create logo.webp

* Create tutorial.yml

* Create 01.webp

* Create fr.md

* img

* Update fr.md

* Update fr.md

* txt OK

* Update fr.md

* img + path

LoicPandul committed:
[TUTORIAL] Merge GitHub tutos contribution (#1480)

* merge tutos

* img path

* img path

* img path

* remove tutorial merged

* fix links

Commit:
Adding Bea event in Honduras (#1485)

* Adding Bea event in Honduras

* Update event.yml

---------

Co-authored-by: Mari <Mari@jjhodl.com>
Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

LoicPandul committed:
[TUTORIAL] Add Bitstack (#1488)

* update the logo of the project

* Create cover.webp

* Create fr.md

* Create tutorial.yml

* txt ok

* Update fr.md

* img + path

BitPreto committed:
[TUTORIAL][RESOURCE] Add Nakamochi (#1489)

* add tutorial, professor, builder nakamochi

* professor, fix nakamochi

* project updated from builder,

LoicPandul committed:
[TUTORIAL] Liana - Remove french videos on translation (#1491)

* Update cs.md

* Update de.md

* Update en.md

* Update es.md

* Update et.md

* Update fi.md

* Update id.md

* Update it.md

* Update nb-NO.md

* Update ja.md

* Update pt.md

* Update ru.md

* Update vi.md

* Update zh-Hans.md

LoicPandul committed:
[TUTORIAL] Passphrase - Add 2 new links (#1492)

* Update cs.md

* Update de.md

* Update en.md

* Update es.md

* Update et.md

* Update fi.md

* Update fr.md

* Update id.md

* Update it.md

* Update ja.md

* Update nb-NO.md

* Update pt.md

* Update ru.md

* Update vi.md

* Update zh-Hans.md

Asi0Flammeus committed:
fix nakamochi

Asi0Flammeus committed:
add missing translations and update proofreading sections

trigger-xyz committed:
link: update some course link (ids)

trigger-xyz committed:
link: update other course link (ids)

trigger-xyz committed:
link: fix other links

Asi0Flammeus committed:
btc204, fix automatic translations

Asi0Flammeus committed:
glossary, remove word without term key

Asi0Flammeus committed:
fix translation typos

Asi0Flammeus committed:
fix translation typos

trigger-xyz committed:
fix: old secu101 link

Asi0Flammeus committed:
Course laissez faire (#1506)

* his204, first FR proofreading

* his204, convert images to webp

* add his204 professor

* his204, rename images to with standard name

* his204, add course.yml and thumbnail

Asi0Flammeus committed:
his204, add final chapters

Asi0Flammeus committed:
his204, add parts

Asi0Flammeus committed:
his204, add uuids

Asi0Flammeus committed:
his204, fix image preview

trigger-xyz committed:
fix: rename builder.yml to project.yml for nakamochi

trigger-xyz committed:
fix: .de translation for nakamochi tutorial

Asi0Flammeus committed:
professor, fix bip39 id

Asi0Flammeus committed:
btc101, fix conclusion tags

Asi0Flammeus committed:
auto-translate, add conclusion tags

Asi0Flammeus committed:
fix minor sync erros

Asi0Flammeus committed:
professors, add missing uuid

LoicPandul committed:
conversion correction (#1509)

LoicPandul committed:
[DOCS] Delete tutorial-creation-guidelines.md (#1465)

* Delete tutorial-creation-guidelines.md

* add link + restore doc

LoicPandul committed:
Update new-tutorial-creation.py (#1495)

LoicPandul committed:
[TUTORIAL] Add Passport (Batch-2) (#1503)

* Create cover.webp

* Create fr.md

* Create tutorial.yml

* Update fr.md

* txt ok

* urgency: 1 / reward: 0

* orthographe

* img

* img path + italique

* Update fr.md

LoicPandul committed:
[TUTORIAL] - Add Envoy + Passport (#1508)

* Create fr.md

* Create tutorial.yml

* Create cover.webp

* Create 02.webp

* Create 03.webp

* Update fr.md

* txt ok

* img

* img paths

LoicPandul committed:
Delete logo.webp (#1510)

jramos0 committed:
New Builders from Breeze Report (#1511)

* thndr builder

* bitnob builder

* fix typo

* bitpay builder

* Blink builder

* Bolckdaemon builder

* coinbase builder

* geyser fund builder

* Imperia gaming builder

* Joltz builder

* lightspark builder

* moneybadger builder

* mercari builder

* Mondo Builder

* namecheap builder

* Osmo Wallet builder

* picknpay builder

* pubkey bar builder

* satsback builder

* zaprite builder

* Fix lightspark image typo

* New type of project: Merchant

* fix Ocean missing logo

* Salvadorean Builders

LoicPandul committed:
remove old tuto (#1512)

networkeffect committed:
Correction francais biz101 (#1514)

* Update fr.md

* Update fr.md

* Update fr.md

* Update fr.md

LoicPandul committed:
[TUTORIAL] Add Proton Wallet (#1515)

* Create cover.webp

* Create logo.webp

* Create tutorial.yml

* Create fr.md

* txt ok

* img

* img paths

Asi0Flammeus committed:
tutorials, add missing translations

Asi0Flammeus committed:
resources, add missing translations

Asi0Flammeus committed:
fix minor sync errors

trigger-xyz committed:
fix: sync issue on envoy id.md

Commit:
Fixing_envoy_and_passport_Indonesian

trigger-xyz committed:
Merge pull request #1522 from PlanB-Network/Fixing_id.md

Fixing_envoy_and_passport_Indonesian

Asi0Flammeus committed:
auto-translate, update requirements

LoicPandul committed:
[TUTORIAL] Add + Edit 4 tutos about tutorial contribution (#1448)

* remove old translations

* Update tutorial.yml

* PBN : ₿

* commit

* tuto github desktop

* img + path tuto github desktop

* general content tuto ok : img + path + links

* github desktop txt update

* Update fr.md

* Create cover.webp

* Create logo.webp

* Update fr.md

* Update fr.md

* Create cover.webp

* Create logo.webp

* Update fr.md

* tuto expert ok

* Update fr.md

* Update fr.md

* orthographe

* tuto github web txt

* img + path

* Update fr.md

* Update fr.md

* Folder + YAML : builder -> project

* change cover.webp on 4 tutos

* Update fr.md

* Update 39.webp

* Title names

* add paolo@pbn

* 2 alternatives tutorials

* list tags paolo -> P₿N

* img change 17.webp and 25.webp

* properties english only keys

* precise: merge the PR on dev

* update 38.webp (add the arrow on "dev")

* Rename tutorials folders + fix links

* urgency: 1 reward: 0

LoicPandul committed:
portfolio -> wallet (#1517)

LoicPandul committed:
[TUTORIAL] Add Phoenix (#1523)

* remove old tuto Phoenix

* Create fr.md

* Create tutorial.yml

* Create cover.webp

* txt ok

* img path

* img

* Create logo.webp

LoicPandul committed:
Update cover.webp (#1524)

LoicPandul committed:
[TUTORIAL] Add Portal (#1526)

* add Project TwentyTwo Devices

* Create cover.webp

* Create tutorial.yml

* Create fr.md

* txt ok

* img

* img path

Arshia-r-m committed:
Fix to issue 1527 (#1528)

LoicPandul committed:
[BTC102] Add section 4 (#1546)

* add section 4 fr.md

* Update fr.md

* translations

BitPreto committed:
[REOSURCE] Edit Profs, Add channels and projects (#1547)

* professors, edits

* resources, new added

* resources, new added

Commit:
Replacing-btc101-images (#1549)

* Replacing-btc101-images

* Changing_small_detail_all_languages

---------

Co-authored-by: Mari <Mari@jjhodl.com>

Commit:
Updating wordcount reward script (#1550)

* Replacing-btc101-images

* updating-word-reward-script

---------

Co-authored-by: Mari <Mari@jjhodl.com>

LoicPandul committed:
[SCU101] Fix quiz (#1552)

* wrong chapter

* remove 4 questions (will do these chapters entirely)

* 5 questions chpt password manager (translation done)

* 77

* 78

* Update fr.yml

* Update fr.yml

* Update fr.yml

* 79

* 80

* 81

BitPreto committed:
Fix nakamochi elysium tutos (#1553)

* tutorials, fix elysium and nakamochi

* Adding_translations_elysium-nakamochi

---------

Co-authored-by: Asi0Flammeus <datascript@ns3104763.ip-145-239-142.eu>
Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

BitPreto committed:
script requirements, added one of many missing packages (#1554)

Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

LoicPandul committed:
[LNP201] Fix ordered lists (#1555)

* Update cs.md

* Update en.md

* Update fr.md

* Update pt.md

* Update fi.md

* Update et.md

* Update es.md

* Update de.md

* Update id.md

* Update ja.md

* Update it.md

* Update nb-NO.md

* Update zh-Hans.md

* Update ru.md

* Update vi.md

Commit:
Update README.md (#1557)

* Update README.md

* Add files via upload

* Add description to image

* Final version

* Last commit

* Update README.md

---------

Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

LoicPandul committed:
remove double <isCourseExam> + <isCourseReview> (#1558)

jramos0 committed:
Biz School 2024 OS version (#1559)

* Biz School 2024 OS version

* biz221, rename and add link

* biz221, add test only flag

---------

Co-authored-by: Asi0Flammeus <asi0.flammeus@pm.me>

jramos0 committed:
New Events (#1566)

* La Casa de Satoshi builder

* La casa de satoshi events

* bienes raices event

* Events with builder created

* Node partner event

* Bitcoin ireland builder and conference

* Barcelona Cyphers 25

* fix on image from event

Asi0Flammeus committed:
auto-translation, add missing translations

Asi0Flammeus committed:
proofreading, update rewards

LoicPandul committed:
Fix links (#1481)

* peach tuto links

* proton mail link in tuto Ubuntu

* remove /fr/ in links

Asi0Flammeus committed:
Replace tags with fixed list choice (#1573)

* delete all tag values

* update all tags based on fixed list

Asi0Flammeus committed:
biz221, update name

Asi0Flammeus committed:
fix translations

BitPreto committed:
fix typo, cyrillic in the text (#1585)

BitPreto committed:
professor, added to the repo (#1606)

Asi0Flammeus committed:
[FIX] ensure empty line after planb.network links

trigger-xyz committed:
fix: wront indentation

Commit:
deleting_event (#1615)

Co-authored-by: Mari <Mari@jjhodl.com>

Commit:
Updating_timestamp_tutorial (#1623)

* Updating_timestamp_tutorial

* Update en.md

* Update en.md

* Update en.md

* Update en.md

---------

Co-authored-by: Mari <Mari@jjhodl.com>

jramos0 committed:
New International events for 2025 (#1625)

* Testing Event online location

* Treasure Summit 2025

* International Bitcoin Events

* International Bitcoin Events fix typos

* International Bitcoin Events fix timezones and schedule

* new builder

Asi0Flammeus committed:
event, folder structure

BitPreto committed:
[RESOURCE] Fix conf replays stages + Add podcast (#1493)

* conf replayes, 1st batch of edits

* conf replays, 2nd batch

* newsletter, fix thumbnail size

* new podcast + fix conference.yml files

* fix "projectId" typos

LoicPandul committed:
[TUTORIAL] Add Sparrow Wallet (#1572)

* remove old tuto

* Update tutorial.yml

* Update fr.md

* Create cover.webp

* Update fr.md

* Update fr.md

* txt ok

* Update fr.md

* img

* img paths

LoicPandul committed:
[TUTORIAL] Bitcoin Core - remove logo.webp (#1574)

* Delete logo.webp

* Delete logo.webp

LoicPandul committed:
[TUTORIAL] Edit Passport + Envoy : Nokia battery (#1575)

* passport tuto

* envoy tuto

LoicPandul committed:
update portal/en.md (#1579)

LoicPandul committed:
Fix links (#1580)

* fix link to proton mail tuto

* fix links basic of github tuto

* double link

LoicPandul committed:
[TUTORIAL] Update logos (#1582)

* update raspiblitz logo

* remove logo braiins pool tuto (->project)

* update liquid logo

* update github logo

* update github desktop logo

* proton mail logo

* proton drive logo

* update payjoin sparrow

* update joinbot logo

* update payjoin sw logo

* update payjoin logo

* update whirlpool logo

* update ricochet logo

* update stonewall x2 logo

* update stonewall logo

LoicPandul committed:
Fix courses links (#1583)

* fix links BTC302 fr.md

* Update course_documentation.md

* fix links in some docs

LoicPandul committed:
[SCRIPT] Update Data Creator (#1587)

* réorganisation de l'app

* GUI ok

* tuto page ok

* tuto page ok

* professor page

* professor page OK !

* Update project_page.py

* Project page ok

* Delete new-tutorial-creation.py

* Update main.py

* Update config.py

* Update constants.py

* Update data_loader.py

* Update file_ops.py

* Update home.py

* Update tutorial_page.py

* Update professor_page.py

* Update project_page.py

* terminé

* Update professor_page.py

jimzap committed:
Profile Picture Jim Modification  (#1590)

* Delete professors/jim/assets/profile.webp

* Create rr

* Add files via upload

* Update and rename rr to rr.md

* Delete professors/jim/assets/rr.md

LoicPandul committed:
[PROJECT] Add Canaan (#1591)

* Create logo.webp

* Create fr.yml

* Create project.yml

LoicPandul committed:
[PROFESSOR] Add Louferlou (#1592)

* Create profile.webp

* Create fr.yml

* Create professor.yml

* Update professor.yml

LoicPandul committed:
[TUTORIAL] Add tuto contribution - create prof profile (#1597)

* remove old tutorial

* Update cover.webp

* Update fr.md

* tuto ok

Commit:
Updating-Weblate-tutorials (#1598)

* Weblate-tutorials-new version

* Update en.md

* Add files via upload

* Update en.md

* Adding image 21

* final version

* Update en.md

* correction

* Adding_translations

* Fix Plan B Network

* Correcting_Plan_B_Network

* Update tutorials/others/weblate-add-new-language/fr.md

Co-authored-by: Loïc <137194052+LoicPandul@users.noreply.github.com>

* Apply suggestions from code review

Co-authored-by: Loïc <137194052+LoicPandul@users.noreply.github.com>

* Apply suggestions from code review

Co-authored-by: Loïc <137194052+LoicPandul@users.noreply.github.com>

---------

Co-authored-by: Mari <Mari@jjhodl.com>
Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>
Co-authored-by: Loïc <137194052+LoicPandul@users.noreply.github.com>

jramos0 committed:
Last modifications and Associated profesors Biz School Previous editions (#1605)

* Changes on Biz School Previous edition

* Changes on profesorrs data Biz School Previous edition

* fix typo on  profesorrs data Biz School Previous edition

BitPreto committed:
project, add BitcoinVoucherBot (#1610)

LoicPandul committed:
[TUTORIAL] Phoenix correction (#1613)

* Update cs.md

* Update de.md

* Update en.md

* Update es.md

* Update et.md

* Update fi.md

* Update fr.md

* Update id.md

* Update it.md

* Update ja.md

* Update nb-NO.md

* Update pt.md

* Update ru.md

* Update vi.md

* Update zh-Hans.md

LoicPandul committed:
[TUTORIAL] Add Bitcoin Knots node (#1617)

* Add Project Bitcoin Knots

* Create tutorial.yml

* Create fr.md

* update folder name -> bitcoin-knots

* Create cover.webp

* Update fr.md

* txt knots ok

* img

* img path

LoicPandul committed:
[TUTORIAL] Fix backticks fr.md contribution tutorial (#1586)

* fix backticks

* remove translations

* adding_en.md

* Adding_other_languages

* new img

* Update fr.md

* Update cs.md

* Update de.md

* Update en.md

* Update es.md

* Update et.md

* Update fi.md

* Update id.md

* Update it.md

* Update ja.md

* Update nb-NO.md

* Update pt.md

* Update ru.md

* Update vi.md

* Update zh-Hans.md

* Update cs.md

* Update de.md

* Update en.md

* Update es.md

* Update et.md

* Update fi.md

* Update id.md

* Update it.md

* Update ja.md

* Update nb-NO.md

* Update pt.md

* Update ru.md

* Update vi.md

* Update zh-Hans.md

* tutorials, fix translations

* tutorials, fix translations

---------

Co-authored-by: Mari <Mari@jjhodl.com>
Co-authored-by: Asi0Flammeus <asi0.flammeus@pm.me>
Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

LoicPandul committed:
[TUTORIAL] Refactor tutorial categories and subcategories v2 (#1628)

* move data tuto

* fix links URL

* tutorial -> content subcategory

Asi0Flammeus committed:
fix format

LoicPandul committed:
update subcategories

trigger-xyz committed:
Merge pull request #1631 from LoicPandul/tutorial-categories-last-modifications

[TUTORIAL] Update subcategories

LoicPandul committed:
fix links sub tutorial -> content in URLs

LoicPandul committed:
Move tutorials Kraken and Bitfinex pro to business/others/ cat

trigger-xyz committed:
Merge pull request #1632 from LoicPandul/tutorial-categories-fix-links

[TUTORIAL] Links corrections + relocation of 2 tutos

Commit:
Updating rewards for EN,DE,PT,FR (#1638)

Co-authored-by: Asi0Flammeus <datascript@ns3104763.ip-145-239-142.eu>

Asi0Flammeus committed:
remove builder field from all yml files

Asi0Flammeus committed:
books, fix contributors value

Asi0Flammeus committed:
replace contributor values by github usernam

Asi0Flammeus committed:
replace contributors field by contributor_names

Asi0Flammeus committed:
replace contributors_id field by contributor_names

Asi0Flammeus committed:
replace contributors_id field by contributor_names in proofreading script

Asi0Flammeus committed:
move contributor_names to content yml file

Asi0Flammeus committed:
replace address_line_1 field by address_city_country

Commit:
Updating_proofreading_tutorial (#1642)

* Updating_proofreading_tutorial

* Update tutorial.yml

* changing title_deleting old tutorial

* changing logo to distinguish this from the tutorial creations files

* changing category

---------

Co-authored-by: Mari <Mari@jjhodl.com>

trigger-xyz committed:
Merge branch 'dev' into replace-contributor-names

trigger-xyz committed:
fix book

trigger-xyz committed:
Merge pull request #1640 from PlanB-Network/replace-contributor-names

Replace contributor names

trigger-xyz committed:
fix tutorial

trigger-xyz committed:
Merge pull request #1641 from PlanB-Network/replace-address-line-1

replace address_line_1 field by address_city_country

Asi0Flammeus committed:
tutorials, remove credits field

Asi0Flammeus committed:
professors, replace bip39 username to uuid in course and tutorials

Asi0Flammeus committed:
professors, replace professors field by professors_id

Asi0Flammeus committed:
professors, replace professor field by professor_id

Asi0Flammeus committed:
replace affilitions by uuids reference of the associated project

Asi0Flammeus committed:
Add new translation features (#1656)

* auto-translate, add non-translable word feature

* auto-translate, add farsi

* auto-translate, add custom prompt

trigger-xyz committed:
fix: credit link on tutorials

trigger-xyz committed:
fix: some professor id in tutorials

trigger-xyz committed:
remove wrong professor_id

Asi0Flammeus committed:
project, add SOB

Asi0Flammeus committed:
project, fix uuid

Asi0Flammeus committed:
project, fix path

Asi0Flammeus committed:
project, add tags

Asi0Flammeus committed:
project, fix logo name

trigger-xyz committed:
Merge pull request #1653 from PlanB-Network/replace-professor-id

Replace professor

trigger-xyz committed:
Merge pull request #1654 from PlanB-Network/replace-affiliations

replace affilitions by uuids reference of the associated project

Commit:
Updating-proofreading-tutorial (#1599)

* Changing-title

* adding_paragraph

* Update de.md

* Update en.md

* Update en.md

* correction

* typo

---------

Co-authored-by: Mari <Mari@jjhodl.com>

LoicPandul committed:
[CSV402] Fix tables form in translations (#1626)

* fix tabs

* Update cs.md

* Update es.md

* Update nb-NO.md

* Update de.md

* Update id.md

* Update et.md

* Update en.md

* Update pt.md

* Update it.md

* Update fi.md

* Update ja.md

* Update ru.md

* Update vi.md

* Update zh-Hans.md

* Update cs.md

* Update de.md

* Update en.md

* Update es.md

* Update et.md

* Update fi.md

* Update id.md

* Update it.md

* Update ja.md

* Update nb-NO.md

* Update pt.md

* Update ru.md

* Update vi.md

* Update zh-Hans.md

* Update cs.md

* Update de.md

* Update en.md

* Update es.md

* Update et.md

* Update fi.md

* Update id.md

* Update it.md

* Update ja.md

* Update nb-NO.md

* Update pt.md

* Update ru.md

* Update vi.md

* Update zh-Hans.md

* Adding_vietnamese_translation

* fix vi.md tabs

---------

Co-authored-by: Mari <Mari@jjhodl.com>

Commit:
Translating_missing_courses_vietnamese (#1629)

Co-authored-by: Asi0Flammeus <datascript@ns3104763.ip-145-239-142.eu>
Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

BitPreto committed:
[CYP302] Fix formula typos (#1630)

* cyp302, fix formulas typos

* cyp302, fix llm typo for it.md

---------

Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

LoicPandul committed:
[TUTORIAL] SBP - Remove french video (#1633)

* Update cs.md

* Update de.md

* Update en.md

* Update es.md

* Update et.md

* Update fi.md

* Update id.md

* Update it.md

* Update ja.md

* Update nb-NO.md

* Update pt.md

* Update ru.md

* Update vi.md

* Update zh-Hans.md

LoicPandul committed:
[TUTORIAL] Bitcoin Core change title (#1635)

* Update title Bitcoin Core (Linux) tuto

* update title Bitcoin Core (macOS & Windows) tuto

LoicPandul committed:
remove BTC204 fr videos on translations (#1636)

LoicPandul committed:
[SCRIPT] Update Data Creator with new categories (#1637)

* Update constants.py

* Update file_ops.py

LoicPandul committed:
Fix links (#1639)

* fix links business/others

* fix links sparrow wallet tuto

* links weblate and github tutos

* Update README.md

---------

Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

Gloirew committed:
Conference Bitcoin Afrique  (#1657)

* Update fr.md

Correction d'une faute d'orthographe

* Create events.yml

New conf in africa

* Create .gitkeep

* Add files via upload

* Add files via upload

* Delete events/bitcoin-conference-afrique-2025/assets/Logo.jpeg

* Delete events/bitcoin-conference-afrique-2025/assets/.gitkeep

* events, fix yaml file and thumbnail file names

---------

Co-authored-by: Asi0Flammeus <asi0.flammeus@pm.me>

BitPreto committed:
[TUTO] Add Satsmobi, BitcoinVoucherBot, Wallet of Satoshi (#1664)

* tutorials, 2 new imports from f205sats

* imported tutorials, fix category

* tutorials, 1 new import from f205sats

* Update tutorial.yml

* Update tutorial.yml

* Update tutorial.yml

* Update tutorial.yml

* Update tutorial.yml

---------

Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

jramos0 committed:
Tutorial Local Environment and Spanish Content (#1665)

* Tutorial Local Deployment of PBN Platform v1

* Professor profile & fix typos

* Adding images to tutorial

* Fix professor id

* fix typo

* Lunaticoin newsletter

* Tesis Bitcoin Newsletter

* BTC en español newsletter

LoicPandul committed:
[CYP201] Fix an error (#1668)

* Update fr.md

* Update cs.md

* Update de.md

* Update en.md

* Update es.md

* Update et.md

* Update fi.md

* Update id.md

* Update it.md

* Update ja.md

* Update nb-NO.md

* Update pt.md

* Update ru.md

* Update vi.md

* Update zh-Hans.md

LoicPandul committed:
[LNP201] Fix an error (#1669)

* Update cs.md

* Update de.md

* Update en.md

* Update es.md

* Update et.md

* Update fi.md

* Update fr.md

* Update id.md

* Update it.md

* Update ja.md

* Update nb-NO.md

* Update pt.md

* Update ru.md

* Update vi.md

* Update zh-Hans.md

Asi0Flammeus committed:
tutorials, fix sync issue

Asi0Flammeus committed:
tutorials, fix sync issue

Asi0Flammeus committed:
auto-translate, add missing automatic translations

BitPreto committed:
[DOCS] Edit formatting rules (#1670)

* Update awesome-tutorial.yml

* docs, tutos categories

* docs, edit dummy tutos and prof

Asi0Flammeus committed:
tutorials, fix typos

trigger-xyz committed:
add first test_only event for testing purpose

trigger-xyz committed:
add 2nd test_only event for testing purpose

trigger-xyz committed:
add 3rd test_only event for testing purpose

trigger-xyz committed:
add 4th test_only event for testing purpose

trigger-xyz committed:
update test events

trigger-xyz committed:
update test events

LoicPandul committed:
[SCRIPT] Update Data Creator App (#1690)

* Update home.py

* Update professor_page.py

* Update constants.py

* Update data_loader.py

* Update file_ops.py

* Update README.md

LoicPandul committed:
[TUTORIAL] Update contribution tutorials guidelines (#1688)

* fix english translation issues

* Update en.md

* Update en.md

* update guidelines and fix translation issues git-expert tuto

* Update it.md

* Update id.md

* Update en.md

* Update de.md

* Update nb-NO.md

* Update fr.md

* Update et.md

* Update es.md

* Update en.md

* Update es.md

* Update pt.md

* Update cs.md

* Update cs.md

* Update zh-Hans.md

* Update de.md

* Update ja.md

* Update id.md

* Update fr.md

* Update nb-NO.md

* Update et.md

* Update ru.md

* Update vi.md

* Update fi.md

* Update it.md

* Update ru.md

* Update vi.md

* Update fi.md

* Update pt.md

* Update it.md

* Update zh-Hans.md

* Update ja.md

* update img screenshots data-creator software

* contribution tutorial prof BIP39 ID -> UUID

Asi0Flammeus committed:
add project_id to content

Asi0Flammeus committed:
auto-translate, fix initial upload script

Asi0Flammeus committed:
auto-translate, add argument for initial upload script

Asi0Flammeus committed:
auto-translate, fix path

Asi0Flammeus committed:
Hindi initial upload (#1725)

* btc101, add hi initial translation

* auto-translate, add argument for initial upload script

* auto-translate, fix path

* update initial upload script

* Adding_Hindi_language

---------

Co-authored-by: Asi0Flammeus <datascript@ns3104763.ip-145-239-142.eu>

Asi0Flammeus committed:
hi, fix project yml

Asi0Flammeus committed:
hi, fix bet

Asi0Flammeus committed:
hi, fix book description

Asi0Flammeus committed:
hi, fix resources yml format

Asi0Flammeus committed:
hi, fix goal format

Asi0Flammeus committed:
hi, fix yml format

Asi0Flammeus committed:
hi, fix yml format

Asi0Flammeus committed:
hi, remove tutorial with bad description formatting

Asi0Flammeus committed:
hi, remove quizzes with bad description formatting

Asi0Flammeus committed:
hi, remove quizzes with bad description formatting

Asi0Flammeus committed:
hi, remove quizzes with bad explanation formatting

Asi0Flammeus committed:
hi, fix indentation quizz

Asi0Flammeus committed:
hi, remove tutorials

Asi0Flammeus committed:
hi, remove professors

Asi0Flammeus committed:
hi, remove quizzes

AlphonseMehounme committed:
Add Bitdevs Cotonou Meetup (#1655)

* Create events.yml for Bitdevs Cotonou Meetup

* Create .gitkeep

* Add files via upload

* Delete resources/conferences/bitdevs-cotonou-meetup-march-2025/assets/.gitkeep

Delete .gitkeep file

* fix: timezone and id added and booking deleted

BitPreto committed:
[RESOURCE] Add Zaprite YT Channel (#1672)

* yt_channel, add zaprite

* project, edit zaprite yml

MilankovicProtocol committed:
[CORRECTION] Corrected a typo (#1678)

BitPreto committed:
[BTC101] Edit fr.md (#1683)

* [BTC101] Edit fr.md

preliminary restore of french YT videos

* btc101, fix typos embedded links

LoicPandul committed:
[TUTORIAL] Fix Sparrow Wallet fr.md (#1686)

* remove bad translations

* fix 4 backticks fr.md

* add_translation_sparrow

* Sparrow tuto en.md - add links to  Southern Bitcoiner's videos

* sparrow tuto en.md - portfolio -> wallet

---------

Co-authored-by: Mari <Mari@jjhodl.com>

LoicPandul committed:
bitcoin knots -> wallet translation fix (#1687)

LoicPandul committed:
[TUTORIAL] Update contribution add a professor (#1689)

* Update 01.webp

* Update 02.webp

* Update fr.md

* Update fi.md

* Update de.md

* Update ja.md

* Update et.md

* Update nb-NO.md

* Update cs.md

* Update id.md

* Update pt.md

* Update zh-Hans.md

* Update es.md

* Update ru.md

* Update it.md

* Update vi.md

* Update en.md

LoicPandul committed:
[BET] Add recovery sheet by Meyga Vox (#1691)

* add BET recovery sheet by Meyga Vox

* Update logo.webp

jramos0 committed:
New Events and ressources (#1706)

* Liquid Bootcamp AR

* Fix address event

* Liquid bootcamp Bg

* liquid bootcamp It

* Fix Timezone It

* Fix Timezone It

* Fix city_country space

* Spanish YT Channel

* ES yt channel btc en espanol

* Es yt channel Bitcoin podcast el salvador

* Bitcoin Hoy es yt channel

jramos0 committed:
Sid302 & sid202 right conclusion format (#1709)

* Change on conclusion chapter sid302

* Changes on Sid202 conclusion

* fix on sid302 and sid202 translation

LoicPandul committed:
improve tuto Trezor Model One (#1692)

LoicPandul committed:
[CYP302] Review FR (#1710)

* champs -> corps

* cryptage -> chiffrement

* fix links

* Update fr.md

* "cryptage" -> "chiffrement"

* fix hook issues

* section 1 ok

* section 2

* groups

* section 4

* section 5

* section 6

* section 7

* section 8 CYP302 fr.md

* Update fr.md

* Update quizz 050 fr.md

* Update quizz 051 fr.md

* Update quizz 052 fr.md

* Update quizz 053 fr.md

* Update quizz 054 fr.md

* Update quizz 056 fr.md

* Update fr.md

* Update course.yml

* New introduction guidelines on fr.md

* intro catchphrase

LoicPandul committed:
Update requirements.txt (#1711)

BitPreto committed:
book, new cover_en for all but ja lang (#1712)

LoicPandul committed:
[TUTORIAL] Update Alby Hub (#1715)

* Update 09.webp

* Update 47.webp

* Update 48.webp

* Create 66.webp

* Update cs.md

* Update de.md

* Update en.md

* Update es.md

* Update et.md

* Update fi.md

* Update fr.md

* Update id.md

* Update it.md

* Update ja.md

* Update nb-NO.md

* Update pt.md

* Update ru.md

* Update vi.md

* Update zh-Hans.md

* Adding_vi_alby_hub

---------

Co-authored-by: Mari <Mari@jjhodl.com>

LoicPandul committed:
[PROJECT] Update Alby (#1716)

* Update project.yml

* Update cs.yml

* Update de.yml

* Update en.yml

* Update es.yml

* Update et.yml

* Update fi.yml

* Update fr.yml

* Update id.yml

* Update it.yml

* Update ja.yml

* Update nb-NO.yml

* Update pt.yml

* Update ru.yml

* Update vi.yml

* Update zh-Hans.yml

Commit:
changing_contributor_property (#1718)

Co-authored-by: Mari <Mari@jjhodl.com>

LoicPandul committed:
[BIZ101] Fix tables (#1728)

* fix cs.md tabs

* fix de.md tabs

* fix es.md tabs

* fix et.md tabs

* fix fi.md tabs

* fix id.md tabs

* it.md fix tabs

* ja.md fix tabs

* nb-NO.md fix tabs

* pt.md fix tabs

* ru.md fix tabs

* vi.md fix tabs

* zh-Hans.md fix tabs

LoicPandul committed:
[BTC101] Add link to BIZ101 (#1729)

* add link to BIZ101

* move link to chapter 5.2

LoicPandul committed:
Fix links (#1736)

* fix all WoS tuto links

* fix links with new categories

* fix links tuto contribution

* fix links to Sparrow tuto

Commit:
Updating_link_biz101 (#1743)

* Updating_link_biz101

* Delete temporary script

---------

Co-authored-by: Mari <Mari@jjhodl.com>

LoicPandul committed:
fix BTC Map chapter fr.md (#1744)

LoicPandul committed:
Remove 7 questions on the intro (#1747)

PaoloFoti commented:
Hello @hyperfree ! Thank you for the new resources, it seems you have pushed the changes inside a very old branch. the target branch should be the `[PlanB-Network:dev](https://github.com/PlanB-Network/bitcoin-educational-content/tree/dev)`.

Could you try a new PR from the dev branch of your forked repo? Just ensure your repo is up to date with the Plan ₿'s Bitcoin Educational Content before pushing your changes!

hyperfree committed:
PlanB-Network:dev

hyperfree commented:
Hey Paolo, I'm using the web editor and not too familiar so I don't want to
waste your time.
I added the folder with the files I updated, hope this helps.

Best


On Tue, Apr 1, 2025 at 11:26 AM Bitpreto ***@***.***> wrote:

> Hello @hyperfree <https://github.com/hyperfree> ! Thank you for the new
> resources, it seems you have pushed the changes inside a very old branch.
> the target branch should be the [PlanB-Network:dev](
> https://github.com/PlanB-Network/bitcoin-educational-content/tree/dev).
>
> Could you try a new PR from the dev branch of your forked repo? Just
> ensure your repo is up to date with the Plan ₿'s Bitcoin Educational
> Content before pushing your changes!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/PlanB-Network/bitcoin-educational-content/pull/1752#issuecomment-2769565546>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AT7W3Q2S344R2HDRGM33NG32XKOZBAVCNFSM6AAAAAB2FIGPVKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDONRZGU3DKNJUGY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
> [image: PaoloFoti]*PaoloFoti* left a comment
> (PlanB-Network/bitcoin-educational-content#1752)
> <https://github.com/PlanB-Network/bitcoin-educational-content/pull/1752#issuecomment-2769565546>
>
> Hello @hyperfree <https://github.com/hyperfree> ! Thank you for the new
> resources, it seems you have pushed the changes inside a very old branch.
> the target branch should be the [PlanB-Network:dev](
> https://github.com/PlanB-Network/bitcoin-educational-content/tree/dev).
>
> Could you try a new PR from the dev branch of your forked repo? Just
> ensure your repo is up to date with the Plan ₿'s Bitcoin Educational
> Content before pushing your changes!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/PlanB-Network/bitcoin-educational-content/pull/1752#issuecomment-2769565546>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AT7W3Q2S344R2HDRGM33NG32XKOZBAVCNFSM6AAAAAB2FIGPVKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDONRZGU3DKNJUGY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>

hyperfree commented:
They aren't newsletters but news-sites, they may or may not operate
newsletters but categorically they are online news papers or magazines.

On Tue, Apr 1, 2025 at 12:39 PM Bitpreto ***@***.***> wrote:

> I see, the fact is that you can't create new resources types. As of now
> these
> <https://github.com/PlanB-Network/bitcoin-educational-content/tree/dev/docs/PBN-template-repo/resources>
> are the only categories you can add items to. But I am updating this
> documentation so you'll be able to have a broader list of categories, but
> you will never be able to add a new category on your own
>
> The three resources in your latest commit seem to be newsletters, you can
> see in this folder
> <https://github.com/PlanB-Network/bitcoin-educational-content/tree/dev/resources/newsletters>
> the structure you shall follow in order to add new newsletters. Regarding
> the id at the beginning of .yml file you need to generate a new uuid_v4
> from whatever source (even a generator on a website)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/PlanB-Network/bitcoin-educational-content/pull/1752#issuecomment-2769794285>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AT7W3Q2M7TISBZJSNB3MIDT2XKXK3AVCNFSM6AAAAAB2FIGPVKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDONRZG44TIMRYGU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
> [image: PaoloFoti]*PaoloFoti* left a comment
> (PlanB-Network/bitcoin-educational-content#1752)
> <https://github.com/PlanB-Network/bitcoin-educational-content/pull/1752#issuecomment-2769794285>
>
> I see, the fact is that you can't create new resources types. As of now
> these
> <https://github.com/PlanB-Network/bitcoin-educational-content/tree/dev/docs/PBN-template-repo/resources>
> are the only categories you can add items to. But I am updating this
> documentation so you'll be able to have a broader list of categories, but
> you will never be able to add a new category on your own
>
> The three resources in your latest commit seem to be newsletters, you can
> see in this folder
> <https://github.com/PlanB-Network/bitcoin-educational-content/tree/dev/resources/newsletters>
> the structure you shall follow in order to add new newsletters. Regarding
> the id at the beginning of .yml file you need to generate a new uuid_v4
> from whatever source (even a generator on a website)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/PlanB-Network/bitcoin-educational-content/pull/1752#issuecomment-2769794285>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AT7W3Q2M7TISBZJSNB3MIDT2XKXK3AVCNFSM6AAAAAB2FIGPVKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDONRZG44TIMRYGU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>
